pkg/cli/idle/idle.go

```diff
@@ -312,14 +313,19 @@ func (o *IdleOptions) calculateIdlableAnnotationsByService(infoVisitor func(reso
 			return err
 		}
 
-		endpoints, isEndpoints := info.Object.(*corev1.Endpoints)
+		endpoints, isEndpoints := info.Object.(*v1.EndpointSlice)
```

Comment:
```suggestion
		slice, isEndpointSlice := info.Object.(*v1.EndpointSlice)
```

?

Yeah, that will require more rewriting, but it makes it clearer